arrows/mvg/sfm_utils.h

```diff
@@ -226,22 +214,18 @@ clean_cameras_and_landmarks(
   int min_landmark_inliers = -1);
 
 /// Return true if the camera is upright
-/*
- * Test if "up" in the image aligns with "up" in the world for each camera.
- * The alignment in this case means that the dot product of these vectors
- * is postive, not that the vectors are equal.
- * Up in the image is the negative Y-axis.  Up in the world defaults to
- * the postive Z-axis, but this is configurable by specifying \p up.
- */
+//  Test if "up" in the image aligns with "up" in the world for each camera.
+//  The alignment in this case means that the dot product of these vectors
+//  is postive, not that the vectors are equal.
+//  Up in the image is the negative Y-axis.  Up in the world defaults to
+//  the postive Z-axis, but this is configurable by specifying \p up.
```

Comment:
It looks like this was intended to be a Doxygen comment.

Reply:
Fixed.  The doxygen is corrected now.